static/styles/left_sidebar.css

```diff
@@ -395,7 +399,7 @@ li.top_left_recent_topics {
     */
 
     &:hover {
-        color: hsl(0, 0%, 0%) !important;
+        color: hsl(0, 0%, 20%) !important;
```

Comment:
There was a slight variation on how the hover on the dark mode should work compared to the light theme mode. Thereby causing issues in the balance between the two theme modes when hovered on the ellipsis icon.

This commit fixed that by using hsl color values to set a balance between both themes.